src/dragonball/src/api/v1/vmm_action.rs

```diff
@@ -539,6 +573,8 @@ impl VmmService {
         // - Some(path), legacy_manager will create_socket_console on that path.
         config.serial_path = machine_config.serial_path;
 
+        config.pci_hotplug_enabled = machine_config.pci_hotplug_enabled;
```

Comment:
Is this fine without feature attributes? e.g. `#[cfg(all(hotplug, host-device))]`?

Reply:
yep, vm config will always have pci_hotplug_enabled parameter, but when that value pass down, if the `#[cfg(all(feature = "hotplug", feature = "host-device"))]` not enabled, Dragonball won't process `pci_hotplug_enabled`